### PR TITLE
Upgrade to SDK 35

### PR DESCRIPTION
**Pull request by cooltey**
The #5149 has been merged and it's time to upgrade to SDK 35 officially.

### CODE REVIEW

**Comment by dbrant** [comment]
The activity that will still need some tweaking is PageActivity (and maybe a couple of others):

image

Notice that the activity bleeds into the status bar at the top, _and_ the navigation bar at the bottom. I believe this is because PageActivity is already "trying" to handle insets correctly, but just needs to be updated to handle top and bottom insets.

**Comment by cooltey** [comment]
> The activity that will still need some tweaking is PageActivity (and maybe a couple of others):
> 
> image
> 
> Notice that the activity bleeds into the status bar at the top, _and_ the navigation bar at the bottom. I believe this is because PageActivity is already "trying" to handle insets correctly, but just needs to be updated to handle top and bottom insets.

Thanks for the tips! I'll make updates for those screens that got affected!

**Comment by cooltey** [comment]
> The activity that will still need some tweaking is PageActivity (and maybe a couple of others):
> Notice that the activity bleeds into the status bar at the top, _and_ the navigation bar at the bottom. I believe this is because PageActivity is already "trying" to handle insets correctly, but just needs to be updated to handle top and bottom insets.

This is fixed. I also fixed the `BottomSheet` presentation, but it is imperfect. The navigation bar color change function does not work properly.

**Review by dbrant — CHANGES_REQUESTED** [review]

File: app/src/main/java/org/wikipedia/page/ExtendedBottomSheetDialogFragment.kt [hunk 1]
```diff
@@ -27,7 +30,12 @@ open class ExtendedBottomSheetDialogFragment : BottomSheetDialogFragment() {
     override fun onStart() {
         super.onStart()
         dialog?.window?.let {
-            DeviceUtil.setNavigationBarColor(it, ResourceUtil.getThemedColor(requireContext(), R.attr.paper_color))
+            DeviceUtil.setNavigationBarColor(it, ResourceUtil.getThemedColor(requireContext(), R.attr.warning_color))
```
[the remark this answers is not in the document]
Comment (cooltey) [reply app/src/main/java/org/wikipedia/page/ExtendedBottomSheetDialogFragment.kt]: 😅  fixed.

**Review by dbrant — APPROVED** [review]

**Pull request merged**